src/functions_math.jl

```diff
@@ -127,3 +132,152 @@ function symmetric_names(L::Tuple{Symbol,Symbol}, dims::Integer)
     end
     return compile_time_return_hack(names)
 end
+
+################################################
+# FFT
+
+for fun in (:fft, :ifft, :bfft, :rfft, :irfft, :brfft)
+    plan_fun = Symbol(:plan_, fun)
+
+    if fun in (:irfft, :brfft) # These take one more argument, a size
```

Comment:
```suggestion
    if fun in (:irfft, :brfft)  # These take one more argument, a size
```

Two spaces before end of line comments, per BlueStyle